### ISSUE TITLE
Stats don't apply when unlock node

### ISSUE THREAD

**Issue by luattruong2908** [Pryzinho/MMOCoreAstraAddon#2]
Hi guy, i'm using your addon now, but I face an issue that the stats I give in my skill node don't apply when I use /rpgst command to unlock it.
![image](https://github.com/user-attachments/assets/dc01c0bd-2e47-4033-bdc9-99bbc902d007)

**Comment by Pryzinho** [comment]
Is the stat applied if you acquire it in the conventional way (without using the command rpgst)?
I don't think it's an issue with the addon; I'll review the code since it's been a few months since I last worked on it. It might be due to an phoenix API update.

**Comment by luattruong2908** [comment]
Yes, my stat applied when i acquire it in the conventional way but not when i set it using rpgst command
